sandbox-web/src/main/java/com/exadel/sandbox/team5/DiscountRestController.java

```diff
@@ -26,7 +30,18 @@ public Discount getDiscount(@PathVariable Long id) {
     }
 
     @PostMapping
-    public Discount save(@RequestBody Discount entity) {
+    public Discount save(@RequestParam("file") MultipartFile file,
+            @RequestBody Discount entity) throws IOException {
+
+        if(file != null && !file.getOriginalFilename().isEmpty()){
+            File upLoadDir = new File(upLoadPath);
```

Comment:
Please create for the that separate spring component.

Reply:
I don't understand what I must do!